profile/profile.yaml

```diff
@@ -0,0 +1,166 @@
+      # Keywords: Java 8, Lambda, Stream API, IntelliJ
+      - "Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+
+# List of personal or academic projects
+# Feel free to add more projects
+# Please read the project description guideline carefully. Make sure you follow the simplified SDLC pattern. (t.ly/NlXQ)
+highlighted_projects:
+    #project name
+  - name: "3-Band Equalizer"
+    #URL is optional
+    git_url: "https://github.com/Plum-Tree-Garden/JUCE/tree/main/3BandEQ"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: Virtual Studio Technology - Low Cut, High Cut and Parametric Equalizer with Frequency and Gain control.
+  - name: "Database Manager"
+    git_url: "https://github.com/Plum-Tree-Garden/JAVA/tree/main/JDBC3"
+    description: Database storage, input and retrieval for Ministry of Health of Ontario vaccine counts. 
+  - name: "Typing Tutor"
+    git_url: "https://github.com/Plum-Tree-Garden/JAVA/tree/main/typingTutor"
+    description: Helps users learn typing without looking at keys. Has onscreen keyboard display and highlights keys that are being pressed 
+  - name: "Weighted Learning - Rock Paper Scissors"
+    git_url: "https://github.com/Plum-Tree-Garden/cpp/tree/main/RockPaperScissors"
+    description: Creates a simulation between 2 AI opponents in a game of Rock, Paper, Scissors; then weighs the results to determine future decisions. 
+  - name: "Quantifying Gravitational Force"
+    git_url: "https://github.com/Plum-Tree-Garden/cpp/tree/main/quantify/GravitationalForce"
+    description: Receive user input then quantifies the Neutrons of Gravitational Force between 2 objects on Earth. 
+
+#List of professional experience
+# Please read the project description guideline carefully. Make sure you follow the simplified SDLC pattern. (t.ly/NlXQ)
+#The description must be more than 25 words. If your experience is not software related, please focus on soft skills and achievements (e.g. team collaboration, problem-solving, improvements, communication).
+professional_experience:
+  - title: "Software Developer"
+    company: "Jarvis"
+    duration: "2021-present"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Serve as a member and lead of the software development team. Facilitate proper design and implementation of software. Provide detailed and innovative design; creating a seamless software experience."
+  - title: "Full-Stack Developer"
+    company: "Freelance - https://stormy-plateau-32984.herokuapp.com/"
+    duration: "2021"
+    description: "Provided Front-End website architecture, user interactions on it's web page and end-to-end service. Also, developed Back-End and managed server functionality; ensuring cross-platform optimization for mobile phones."
```

Comment:
please mention the tech stack.
e.g. front-edn with HTML/CSS/JS/Angular and back-end with ....